e2e/cypress/integration/enterprise_saml/okta_login_spec.js

```diff
@@ -1,9 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import users from '../../../fixtures/saml_users.json';
+import users from '../../fixtures/saml_users.json';
+
+/**
+ * Note:
+ * 1. Requires Cypress config for chromeWebSecurity to be false. Could be setup by `export CYPRESS_chromeWebSecurity=false`.
```

Comment:
Wasn't aware that we can override a config top-level field using an env variable - the Cypress documentation mentions being able to do that for "env" section fields - really helpful!  Maybe move Note #1 to the saml-okta-setup.txt file?

Reply:
Hello Catalin, well noted and thanks.  I'll get back into this once https://github.com/mattermost/mattermost-webapp/pull/5107 is merged.